source/slang-capture-replay/replay/decoder-helper.cpp

```diff
@@ -0,0 +1,53 @@
+#include <vector>
```

Comment:
Since we have `List`, we shouldn't use `<vector>`.

Reply:
act